content/en/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md

```diff
@@ -137,28 +136,23 @@ written to a new temporary directory, and the `..data` symlink is updated
 atomically using
 [rename(2)](http://man7.org/linux/man-pages/man2/rename.2.html).
 
-{{< note >}}
-**Note:** A container using Downward API as a
-[subPath](/docs/concepts/storage/volumes/#using-subpath) volume mount will not
-receive Downward API updates.
-{{< /note >}}
-
 Exit the shell:
 
 ```shell
-/# exit
+exit
+/
```

Comment:
This line with a `/` is unnecessary and will cause an error as that's not a command.